src/libsync/vfs/cfapi/cfapiwrapper.cpp

```diff
@@ -146,6 +148,23 @@ void CALLBACK cfApiFetchDataCallback(const CF_CALLBACK_INFO *callbackInfo, const
         return;
     }
 
+    QLocalSocket cancellationSocket;
```

Comment:
please use something like `signalSocket` to allow later (if needed) to send something else apart from cancellation messages